README.md

```diff
@@ -82,92 +76,16 @@ workbook.add_worksheet do |sheet|
   sheet.add_border 'B2:D5'
   sheet.add_border 'B3:D3', [:top]
 end
-workbook.apply_styles
 axlsx.serialize 'grocery.xlsx'
 ```
 
-### `axlsx` gem without `axlsx_styler`
-
-Whith plain `axlsx` you need to know which styles you're going to use beforehand.
-The code for our example is a bit more envolved:
-
-```ruby
-require 'axlsx'
-axlsx = Axlsx::Package.new
-wb = axlsx.workbook
-border_color = '000000'
-wb.add_worksheet do |sheet|
-  # top row
-  header_hash = { b: true, bg_color: '95AFBA' }
-  top_left_corner = wb.styles.add_style header_hash.merge({
-    border: { style: :thin, color: border_color, edges: [:top, :left, :bottom] }
-  })
-  top_edge = wb.styles.add_style header_hash.merge({
-    border: { style: :thin, color: border_color, edges: [:top, :bottom] }
-  })
-  top_right_corner = wb.styles.add_style header_hash.merge({
-    border: { style: :thin, color: border_color, edges: [:top, :right, :bottom] }
-  })
-  sheet.add_row
-  sheet.add_row(["", "Product", "Category", "Price"],
-    style: [ nil, top_left_corner, top_edge, top_right_corner ]
-  )
-
-  # middle rows
-  color_hash = { bg_color: 'E2F89C' }
-  left_edge = wb.styles.add_style color_hash.merge(
-    b: true,
-    border: {
-      style: :thin, color: border_color, edges: [:left]
-    }
-  )
-  inner = wb.styles.add_style color_hash
-  right_edge = wb.styles.add_style color_hash.merge(
-    alignment: { horizontal: :left },
-    border: {
-      style: :thin, color: border_color, edges: [:right]
-    }
-  )
-  sheet.add_row(
-    ["", "Butter", "Dairy", 4.99],
-    style: [nil, left_edge, inner, right_edge]
-  )
-  sheet.add_row(
-    ["", "Bread", "Baked Goods", 3.45],
-    style: [nil, left_edge, inner, right_edge]
-  )
-
-  # last row
-  bottom_left_corner = wb.styles.add_style color_hash.merge({
-    b: true,
-    border: { style: :thin, color: border_color, edges: [:left, :bottom] }
-  })
-  bottom_edge = wb.styles.add_style color_hash.merge({
-    border: { style: :thin, color: border_color, edges: [:bottom] }
-  })
-  bottom_right_corner = wb.styles.add_style color_hash.merge({
-    alignment: { horizontal: :left },
-    border: { style: :thin, color: border_color, edges: [:right, :bottom] }
-  })
-  sheet.add_row(["", "Broccoli", "Produce", 2.99],
-    style: [nil, bottom_left_corner, bottom_edge, bottom_right_corner]
-  )
-
-  sheet.column_widths 5, 20, 20, 20
-end
-axlsx.serialize "grocery.xlsx"
-```
-
-## Installation
-
-Add this line to your application's Gemfile:
-
-    gem 'axlsx_styler'
-
-And then execute:
+Producing the same spreadsheet with vanilla `axlsx` turns out [a bit trickier](./examples/vanilla_axlsx.md).
 
-    $ bundle
 
-Or install it yourself as:
+## Change log
 
-    $ gem install axlsx_styler
+Version | Change
+--------|-------
+0.1.4 | Hide `workbook.apply_styles` method to make it usage easier.
```

Comment:
Spelling's wrong. Should be "Hide `Workbook#apply_styles` method to make easier to use".